packages/fuels/tests/types_scripts.rs

```diff
@@ -167,32 +167,34 @@ async fn main_function_vector_arguments() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test]
-async fn test_script_raw_slice() -> Result<()> {
-    setup_program_test!(
-        Wallets("wallet"),
-        Abigen(Script(
-            name = "BimBamScript",
-            project = "packages/fuels/tests/types/scripts/script_raw_slice",
-        )),
-        LoadScript(
-            name = "script_instance",
-            script = "BimBamScript",
-            wallet = "wallet"
-        )
-    );
-
-    let raw_slice = RawSlice(vec![40, 41, 42]);
-    let wrapper = Wrapper {
-        inner: vec![raw_slice.clone(), raw_slice.clone()],
-        inner_enum: SomeEnum::Second(raw_slice),
-    };
-
-    let rtn = script_instance.main(10, wrapper).call().await?.value;
-    assert_eq!(rtn, RawSlice(vec![1, 2, 3]));
-
-    Ok(())
-}
+// TODO: Decide what to do with this test once
+// https://github.com/FuelLabs/sway/issues/5145 is resolved
+// #[tokio::test]
+// async fn test_script_raw_slice() -> Result<()> {
+//     setup_program_test!(
+//         Wallets("wallet"),
+//         Abigen(Script(
+//             name = "BimBamScript",
+//             project = "packages/fuels/tests/types/scripts/script_raw_slice",
+//         )),
+//         LoadScript(
+//             name = "script_instance",
+//             script = "BimBamScript",
+//             wallet = "wallet"
+//         )
+//     );
+
+//     let raw_slice = RawSlice(vec![40, 41, 42]);
+//     let wrapper = Wrapper {
+//         inner: vec![raw_slice.clone(), raw_slice.clone()],
+//         inner_enum: SomeEnum::Second(raw_slice),
+//     };
+
+//     let rtn = script_instance.main(10, wrapper).call().await?.value;
+//     assert_eq!(rtn, RawSlice(vec![1, 2, 3]));
+
+//     Ok(())
+// }
```

Comment:
```suggestion
// TODO: Decide what to do with this test once
// https://github.com/FuelLabs/sway/issues/5145 is resolved
 #[tokio::test]
 #[ignore]
 async fn test_script_raw_slice() -> Result<()> {
     setup_program_test!(
         Wallets("wallet"),
         Abigen(Script(
             name = "BimBamScript",
             project = "packages/fuels/tests/types/scripts/script_raw_slice",
         )),
         LoadScript(
             name = "script_instance",
             script = "BimBamScript",
             wallet = "wallet"
         )
     );

     let raw_slice = RawSlice(vec![40, 41, 42]);
     let wrapper = Wrapper {
         inner: vec![raw_slice.clone(), raw_slice.clone()],
         inner_enum: SomeEnum::Second(raw_slice),
     };

     let rtn = script_instance.main(10, wrapper).call().await?.value;
     assert_eq!(rtn, RawSlice(vec![1, 2, 3]));

     Ok(())
 }
```

We generally just `#[ignore]` the tests we want to skip.

Reply:
Unfortunately, we have to comment out the test. `#[ignore]` will not run it however it will still be compiled and we will get a compile error from the macro.

Reply:
oops sorry then :innocent: